.editorconfig

```diff
@@ -0,0 +1,16 @@
+# top level config
+root = true
+
+# All files:
+# utf-8
+# Unix-style newlines
+# Remove end-of-file newlines
+# Soft indents (2 spaces)
+# Trim tailing whitespace
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = false
```

Comment:
Actually, I reviewed the editorconfig I usually use and I go back on what I said. I prefer true as well 😄